scenarioo-server/src/test/resources/org/scenarioo/business/diffViewer/build1_build.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Wouldn't it be better to name the build according to what it is? Like successfull_ build.xml or so?

Reply:
It was a conscious decision to number these files, because there are 3 builds which have a chronological order and all three builds are used in the test.